cardano-testnet/test/cardano-testnet-test/files/golden/queries/tipOut.json

```diff
@@ -0,0 +1,10 @@
+{
+    "block": "<redacted>",
+    "epoch": "<redacted>",
+    "era": "Conway",
+    "hash": "<redacted>",
+    "slot": "<redacted>",
+    "slotInEpoch": "<redacted>",
+    "slotsToEpochEnd": "<redacted>",
+    "syncProgress": "100.00"
```

Comment:
I'm wondering if assuming `syncProgress` of 100 will lead to test flakiness.

Reply:
I'd say let's see, and if it happens; let's correct it :+1: